lib/codecs/CryptosuiteEncoder.js

```diff
@@ -26,7 +26,14 @@ export class CryptosuiteEncoder extends CborldEncoder {
     if(typeof value !== 'string') {
       return false;
     }
-    const {appContextMap} = transformer;
-    return new CryptosuiteEncoder({value, appContextMap});
+    const {stringTable, typedLiteralTable} = transformer;
+    if(stringTable.has(value)) {
+      return new CryptosuiteEncoder({value, compressionMap: stringTable});
+    } else if(typedLiteralTable.get('https://w3id.org/security#cryptosuiteString').has(value)) {
+      return new CryptosuiteEncoder({
+        value,
+        compressionMap: typedLiteralTable.get('https://w3id.org/security#cryptosuiteString')
+      });
+    }
```

Comment:
Style is to return early and not then `else if`, just `if`. Note the pattern here where the table is repassed and the lookup will just be repeated -- we should just pass the result from the table instead.

We should make this a constant at the top of the file: `'https://w3id.org/security#cryptosuiteString'` -- well, actually, I think it should be a constant somewhere else where the "built-in" `typedLiteralTable` for legacy mode is used. That way there's nothing special to check here. In fact -- maybe we don't even need this codec if we do that?

```suggestion
    if(stringTable.has(value)) {
      return new CryptosuiteEncoder({value, compressionMap: stringTable});
    }
    if(typedLiteralTable.get('https://w3id.org/security#cryptosuiteString').has(value)) {
      return new CryptosuiteEncoder({
        value,
        compressionMap: typedLiteralTable.get('https://w3id.org/security#cryptosuiteString')
      });
    }
```

Reply:
correct, this codec is currently not used. Leaving optimization pending more discussion.

Reply:
If this isn't needed anymore, let's go ahead and remove it.